_lifeofinternatpuppet/post_24.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Week 24
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 24
+---
+
+Hello all,
+
+- On the triage day, I paired up with Ciaran and Paula to help with major release bumps; RHEL 5.x removal, Puppet 5 removal and module release. The module versions were incremented by 1 for example puppetlabs-concat v.7.0.0 to    v8.0.0.
+Check an example [here](https://github.com/puppetlabs/puppetlabs-websphere_application_server/pull/218/files)
+Check the ticket [here](https://tickets.puppetlabs.com/browse/IAC-1482)
+
+- I raised the following PRs for 'Update major dependencies prior to release'
+   - [puppetlabs-wsus_client](https://github.com/puppetlabs/puppetlabs-wsus_client/pull/153)
+   - [puppetlabs-websphere_application_server](https://github.com/puppetlabs/puppetlabs-websphere_application_server/pull/218)
+   - [puppetlabs-tomcat](https://github.com/puppetlabs/puppetlabs-tomcat/pull/433)
+
+- I also spent some time understanding how to do if statements in Ruby, links given below.
+   - [Ruby in Twenty Minutes](https://www.ruby-lang.org/en/documentation/quickstart/)
+   - [Ruby Tutorial For Complete Beginners: Learn Ruby Now!](https://www.rubyguides.com/ruby-tutorial/ruby-if-else/)
+   - [Learn Ruby the Hard Way](https://learnrubythehardway.org/book/)
+
+- I was contacted by the DIO Team regarding this [PR](https://github.com/puppetlabs/puppetlabs-modules/pull/10183) raised before the Christmas break. I paired up with Sheena, the issue was resolved and the PR was merged by the DIO Team. Thanks to Sheena and Jack for a follow up!
```

Comment:
This is a private repo please remove.

Reply:
The paragraph is removed, check the files of the latest commit message pushed to this pr